feat: depot blog post (#3529)
biomejs/website: src/content/docs/blog/depot-exclusive-sponsor.md

  our infrastructure by feeding the benchmarks very complex inputs.
- A release might have required up to ~45 minutes.
- At some point, Biome on Windows couldn't compile due to the memory required, which exceeded the memory available in the GitHub Actions runners.

## Enters Depot

I discovered Depot by chance on socials. A person thanked them because they sponsored their Rust project. So I took a chance and asked Depot if they were willing to sponsor Biome, and [Jacob Gillespie](https://github.com/jacobwgillespie)—CTO and Founder at Depot was very responsive and happy to back Biome up. In fact, Depot use Biome internally for their projects.

Depot started sponsoring Biome's GitHub infrastructure months ago as a gold sponsor. With their support, **our workflows became way faster**:

- Linting for pull requests take around ~2 minutes.
- Tests for pull request take around ~4 minutes.
- Benchmarks for pull requests take around up to ~15 minutes.
- Releases take around ~25 minutes, and memory-related issues are a problem of the past.

## Depot's impact on the ecosystem

> These aren't just numbers, and I tell why. Having a faster CI has had **positive repercussions** to maintainers, contributors and end-users.

Reviewer: nit: "repercussions" has a negative connotation
nit: other wording
```suggestion
These aren't just numbers. Having a faster CI has had a significant **positive impact** to maintainers, contributors and end-users.
```

- [Help](https://github.com/biomejs/biome/blob/main/crates/biome_parser/CONTRIBUTING.md) [building](https://github.com/biomejs/biome/tree/main/crates/biome_yaml_parser) [Biome](https://github.com/biomejs/biome/tree/main/crates/biome_html_parser) [parsers](https://github.com/biomejs/biome/tree/main/crates/biome_markdown_parser)!
  One interesting fact about Biome parsers is that they are recoverable parsers [error resilient](/internals/architecture/#parser-and-cst) which emit a [CST](https://en.wikipedia.org/wiki/Parse_tree) instead of a classic AST.
- Implement new capabilities in our [LSP (Language Server Protocol)](https://github.com/biomejs/biome/tree/main/crates/biome_lsp), or add new features in one of our editor extensions: [VS Code](https://github.com/biomejs/biome-vscode), [Zed](https://github.com/biomejs/biome-zed) and [JetBrains IntelliJ](https://github.com/biomejs/biome-intellij).

#### Financial help

If you believe in the future of the project, you can also help with a financial contribution, via [Open Collective](https://opencollective.com/biome) or [GitHub Sponsors](https://github.com/sponsors/biomejs).

> Additionally, the project provides an [enterprise support program ](/enterprise) where a company you can employ one of the core contributors to work a specific aspect of the Biome toolchain.

Reviewer: nit: whitespace
```suggestion
Additionally, the project provides an [enterprise support program](/enterprise) where a company you can employ one of the core contributors to work a specific aspect of the Biome toolchain.
```
